src/dsp_tools/commands/xmlupload/resource_create_client.py

```diff
@@ -444,7 +427,14 @@ def _transform_text_prop(
     for val in prop.values:
         match val.value:
             case str():
-                values.append(_transform_into_serialise_value(val, permissions_lookup, SerialiseSimpletext))
+                values.append(
```

Comment:
I prefer to keep the text value separate, because not only do we transform, but there are also two different serialiser routes. I think this will make it too complicated to do nicely. But we can always change it in the future...